readthedocs/core/management/commands/update_repos.py

```diff
@@ -71,6 +71,48 @@ def handle(self, *args, **options):
                             state='triggered',
                         )
 
+                        # pylint: disable=no-value-for-parameter
+                        tasks.update_docs_task(
+                            version.project_id,
+                            build_pk=build.pk,
+                            version_pk=version.pk,
+                        )
+                elif version == 'internal':
```

Comment:
These options should be documented somewhere. Also we should be using constants for these, not strings `INTERNAL = 'internal'` in `projects/constants.py` or similar.

Reply:
Okay I'll Add them in the constants. Do we Document Management Commands Somewhere?

Reply:
Not currently -- perhaps we should do something like this: https://docs.readthedocs.io/en/latest/guides/feature-flags.html

Reply:
Okay i'll Add these in another PR.